docs/downstream.md

```diff
@@ -0,0 +1,109 @@
+are following the same annotation format to accomodate VCF files generated by Sarek.
+
+## Calculating RankScores with [Pathfindr](https://github.com/NBISweden/pathfindr)
+
+There are many approaches to score variants, Pathfindr considers many statistics found in the VCF file, collates
+[snpEff](http://snpeff.sourceforge.net/) and [VEP](https://www.ensembl.org/info/docs/tools/vep/index.html) annotations
+to calculate a single score that is written into a HTML page, also into a CSV file. This CSV can be processed with the
+`scripts/addPFRanksToVCF.py` python3 script. The CSV do not have to be generated by Pathfindr: until you have a
+comma-delimited file with a header with the `ID` in the first column and further columns named as `chr`,`start` and
+`rank_score`, the VCF will be annotated with the provided rank score in the CSV:
+
+```text
+ID,sample,rank_score,rank_terms,LOH,whatever,Consequence,IMPACT,SomethingIrrelevant,chr,start,end,width
+chr1:47248600_A/G,SampleTheBig,2,T1_gene,Y,0.926928,MODIFIER,0,0.853685,chr1,47248600,47248600,1
+chr1:110337540_C/T,SampleTheBig,2,T1_gene,Y,0.920036,MODIFIER,0,0.674548,chr1,110337540,110337540,1
+rs1205765240;rs4067337,SampleTheBig,2,T1_gene,"",0.605296,MODIFIER,0.348,0.209617,chr1,148963716,148963717,2
+chr1:197775910_G/A,SampleTheBig,2, TFBS not_canonical,"",0.813335,MODIFIER,0,0.471831,chr1,197775910,197775910,1
+rs142472845,SampleTheBig,2,T2_gene,"",0.519579,MODIFIER,0.002,0.140411,chr1,248458340,248458340,1
+chr2:15453359_C/T,SampleTheBig,2,T2_gene,"",0.514598,MODIFIER,0,0.00526232,chr2,15453359,15453359,1
+chr2:29413588_G/A,SampleTheBig,2,T1_gene,"",0.00738279,MODIFIER,0,0.994607,chr2,29413588,29413588,1
+chr2:70181162_C/T,SampleTheBig,2, high_impact,"",0.201275,HIGH,0,0.76965,chr2,70181162,70181162,1
+rs1335878992,SampleTheBig,2,T2_gene,"",0.00478691,MODIFIER,0,0.66847,chr2,80397367,80397367,1
+rs56269417,SampleTheBig,2,T2_gene,"",0.76779,MODIFIER,0,0.378918,chr2,98265733,98265733,1
+...
+```
+
+The basis of the annotation is `chr:startposition`: all variants in the VCF with the same coordinates will get the same
+RankScore irrespectively of the ALT allele. If there are multiple scores for a single genomic coordinate in the CSV, the
+very last one in the file will be stored only.
+
+## Usage of `scripts/addPFRanksToVCF.py` : transferring Pathfindr rank_score to Scout
+
+Usage is:
+
+```bash
+$ ./addPFRanksToVCF.py -h
+Usage: addPFRanksToVCF.py [OPTIONS]
+
+Options:
+  -v, --vcf TEXT     VCF file to annotate  [required]
+  -c, --csv TEXT     CSV file with ranked scores  [required]
+  -f, --family TEXT  family ID as expected in scout yaml file (usually sample
+                     name of the normal)  [required]
+  -h, --help         Show this message and exit.
+```
+
+Three parameters have to be provided, `-v` and `-c` are the name of the VCF and CSV files respectively, the `-f`
+parameter is the name of the normal sample in the VCF file. MuTect2 (GATK version < 4.0) VCF has a header line like:
```

Comment:
```suggestion
parameter is the name of the normal sample in the VCF file. Mutect2 (GATK version < 4.0) VCF has a header line like:
```

Reply:
Do we still use this version?

Reply:
- for GATK 3.8 mutect is called MuTect2
 - for GATK 4.x mutect is called Mutect2

We do not use GATK 3.8, but I we still have data already analysed by old MuTect2.